lego/apps/events/views.py

```diff
@@ -57,12 +57,21 @@ def get_queryset(self):
             queryset = Event.objects.all()
         return queryset
 
+    def user_should_see_regs(self, event, user):
+        return event.get_possible_pools(user, future=True, is_admitted=False).exists() or \
```

Comment:
Did you check how this affects number of queries?

Reply:
Not sure if my numbers are correct, but I got the same number of queries (35) both when running this and when running master. Retrieving when not authenticated results in 11 queries. It sounds odd that the numbers are equal, so I'm not sure what to believe.